repos.md

```diff
@@ -0,0 +1,16 @@
+- scalacenter/bloop
+- scalacenter/bloop-config
+- scalacenter/effective-scala-slide-code
+- scalacenter/library-example
+- scalacenter/sbt-eviction-rules
+- scalacenter/sbt-scalafix
+- scalacenter/sbt-scalafix-example
+- scalacenter/sbt-version-policy
+- scalacenter/scala3-migrate
+- scalacenter/scaladex
+- scalacenter/scalafix
+- scalacenter/scalafix.g8
+- scalacenter/scalajs-bundler
+- scalacenter/scala-3-migration-guide
```

Comment:
I don’t know if we still maintain this repo since the content has been included in docs.scala-lang. @adpi2, should we have the Steward here?

Reply:
I'll merge now just to see if this all works. If you want to remove the migration guide, we can do it after, or even just edit the `repos.md` @adpi2 .